app/webrtc.go

```diff
@@ -74,6 +86,29 @@ func GetWebrtcToken(sessionId string) (string, *model.AppError) {
 	return token, nil
 }
 
+func GetKopanoWebmeetingsWebrtcToken(sessionId string) (string, *model.AppError) {
+	data := make(map[string]string)
+	data["type"] = "Token"
+	data["id"] = sessionId
+
+	rq, _ := http.NewRequest("POST", *utils.Cfg.WebrtcSettings.GatewayAdminUrl+"/api/v1/admin/auth/tokens", strings.NewReader(model.MapToJson(data)))
```

Comment:
can we make this URL configurable instead of hard coded? worries me that the route can be deprecated or something and we'll need to change it in code instead of a simple setting

Reply:
Yes we could as for now `GatewayAdminUrl` is only used here. As that value is more of a prefix URL and the sub path to a specific API endpoint will never change unless there is a new API version which then in turn most likely also requires changes in the model, i think its fine to have the endpoint path hardcoded. If you still prefer to have it as part of the `GatewayAdminUrl` configuration please let me know and i make the change.